stripe/src/main/java/com/stripe/android/view/PaymentMethodsActivity.java

```diff
@@ -0,0 +1,363 @@
+package com.stripe.android.view;
+
+import android.content.Context;
+import android.content.Intent;
+import android.graphics.drawable.Drawable;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.annotation.VisibleForTesting;
+import android.support.transition.Fade;
+import android.support.transition.TransitionManager;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.support.v7.widget.Toolbar;
+import android.text.TextUtils;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.FrameLayout;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+
+import com.stripe.android.CustomerSession;
+import com.stripe.android.R;
+import com.stripe.android.model.Customer;
+import com.stripe.android.model.CustomerSource;
+
+import java.util.List;
+
+public class PaymentMethodsActivity extends AppCompatActivity {
+
+    public static final String EXTRA_SELECTED_PAYMENT = "selected_payment";
+    static final String EXTRA_PROXY_DELAY = "proxy_delay";
+    private static final String EXTRA_CUSTOMER = "customer";
```

Comment:
I had this odd idea that you might want to use this activity locally just to have a payment selection, but if that's a tool that we want to ship, I should just make a widget. I'll remove :)